packages/material-ui-lab/src/AvatarGroup/AvatarGroup.js

```diff
@@ -51,6 +54,14 @@ const AvatarGroup = React.forwardRef(function AvatarGroup(props, ref) {
           },
         });
       })}
+      {extraAvatars ?
+        <Avatar
+          className={classes.avatar}
+          style={{
```

Comment:
I would rather see this as `classNames`. We can create multiple classNames for the different `margin` values and apply them conditionally.

Reply:
This doesn't directly relate to the pull request. I believe it's a concern for #19761.

Reply:
This was just moved? Then we should address this later.

Reply:
Yes, I think that these lines are the logical extension of #19761. +1 for considering it into a different effort.